composables/use-issues-by-column.ts

```diff
@@ -0,0 +1,40 @@
+import type { ColumnTitle } from '~/constants/columns'
+import type { IssueStatus } from '~/constants/issues'
+import type { Issue } from '~/types/issues'
+
+function filterIssuesWithoutStatus(issues: Issue[]): Issue[] {
```

Comment:
This function was never meant to be a composable. That's why it isn't exported or starts with `use`.

This is just one-off helper used in the composable. The reason I put it in the same file is because I didn't foresee it being used anywhere else. If I ever need to delete this composable, then the function gets deleted with it and there's no risk of it remaining in the codebase unused.

Maybe I should have defined it inside the composable? (Defining functions outside of components/hooks whenever possible is a habit of from React)